### ISSUE TITLE
Android auto: no episodes / stuck on loading episodes 

### ISSUE THREAD

**Issue by Whatsek** [advplyr/audiobookshelf-app#832]
### Steps to reproduce

1.update server to 2.3.x
2.check if android app works, and episodes are there. 
3. Start android auto: select library, see podcasts list, select podcast: stuck on loading episodes or no episodes. 

### Expected behaviour
- Tell us what should happen
See the same episodes as in the android app on the phone

### Actual behaviour
- Tell us what happens
All podcasts are there, but no episodes. If you select a podcast, screen is stuck on: loading episodes 

When connected to android auto, I select an episode from the phone and press play, it plays on android auto. Progress is saved, but next time in the car, when you press resume in android auto, nothing happens. 

### Environment data

Audiobookshelf Version: 2.3.3 server, and latest android app. 

- [x ] Android App?
- [ ] iOS App?

#### Android Issue
Android version:

Device model: Nothing phone

Stock or customized system: Stock

#### iOS Issue
iOS Version:

iPhone model:

**Comment by advplyr** [comment]
It's helpful to put the android app version instead of saying latest because of referencing this in the future

**Comment by Whatsek** [comment]
> It's helpful to put the android app version instead of saying latest because of referencing this in the future

Done! Thank for explaining why!

**Comment by lhassell** [comment]
Can confirm a similar issue. Audiobooks seem to play with no issue, but podcast episodes will not populate unless I open them on my phone first.

**Comment by Whatsek** [comment]
> Can confirm a similar issue. Audiobooks seem to play with no issue, but podcast episodes will not populate unless I open them on my phone first.

Indeed, I only use podcasts, but as a test I imported an audiobook, thats working with no issues. So the problem is only podcast related.

**Comment by duplaja** [comment]
Can confirm this as well. 

Server: v2.3.3

App: 0.9.65-beta

**Comment by andonevris** [comment]
Also having this issue

Server 2.3.3
App 0.9.65-beta

Android 10

**Comment by advplyr** [comment]
This will be fixed in the next release

**Comment by Whatsek** [comment]
Received an Android app update today, but unfortunately it is not completely fixed. I can see episodes now in android Auto, but they wont start playing. So I can now select a podcast, and the episodes are showing. (They didnt show before this update). But when I select an episode, it keeps loading, but does not start playing. Thank you for your time!

**Comment by duplaja** [comment]
Can confirm the issue above. Podcasts are browsible, but not playable.

Audio books play fine when started from Android Auto, but clicking on a podcast episode just loads continually, or if you had already started a book, returns to the book.

I can still play individual episodes by manually starting the podcast via the phone screen first.

**Comment by Whatsek** [comment]
Updated server to  2.5.0, issue is still there. 
If ABS is started and playing on the phone, and then connected to android auto, pause and resume works. So only the initial start of an episode fails to start.

**Comment by advplyr** [comment]
This is fixed for the next release coming sometime this week/weekend

**Comment by duplaja** [comment]
Thank you!

**Comment by Whatsek** [comment]
> This is fixed for the next release coming sometime this week/weekend

Can confirm that this is fixed! Thank you! Very happy podcast-listener in the car again!

**Comment by advplyr** [comment]
Fixed in [v0.9.68-beta](https://github.com/advplyr/audiobookshelf-app/releases/tag/v0.9.68-beta)
